metadata/metadata_test.go

```diff
@@ -26,6 +26,66 @@ import (
 
 var _ = Describe("Metadata", func() {
 
+	When("HasAnnotation/ HasAnnotationWithValue is called", func() {
+		It("should return true when the annotation with/ without the given value is found", func() {
```

Comment:
This `with/without` would disappear after splitting in two. More legible.

Reply:
Updated, thanks!